### ISSUE TITLE
Scrobble timestamp offset from Last.fm value by +8hrs

### ISSUE THREAD

**Issue by kubu4** [benfoxall/lastfm-to-csv#17]

**Comment by davidomarf** [comment]
Why is this an issue?
If you mean that the API timestamp doesn't fit with the time you scrobbled something, it's because the scrobbles are registered in UTC.
If you are using JS, you can create a Date, and then use `.setTime()` sending the timestamp. That will handle the offset adapting it to your timezone.

**Comment by kubu4** [comment]
I point this out because the interface for obtaining a CSV file of Scrobble history is extremely user-friendly. It's a bummer that the data retrieved does not match the data displayed on Last.fm. Just seems like a small oversight, given how simple and easy the interface is for using the Last.fm to csv website.

In regards to your suggestion on how to resolve this, I have little-to-no programming experience, so I (unfortunately) don't understand your suggested method to fix the timestamps.

**Comment by ajayyy** [comment]
@kubu4 I don't think you are understanding what they are saying. The times in the CSV file are in UTC (GMT), so you must convert that time to your time zone. The times are correct, just not in your time zone.

**Comment by kubu4** [comment]
@ajayyy I understand. I just don't know how to make the adjustment. Plus, I figured that the time zone info could also be retrieved from the Last.fm profile.

**Comment by ajayyy** [comment]
@kubu4 Okay, sorry then. Timezone's are a complex thing in programming and are easy to mess up. That is why times are usually given out in UTC.

**Comment by VeXHarbinger** [comment]
It wouldn't be a part of the profile, but a part of the scrobble.  At least in a perfect world.  Doesn't matter where you live, but where you listened to it
